docs/memorystore.md

```diff
@@ -1,15 +1,19 @@
 # Memorystore (redis) + OnlineBoutique
 
-This guide contains instructions for overriding the default in-cluster `redis` database for `cartservice` with Memorystore (redis).
+This guide contains instructions for overriding the default in-cluster `redis`
+database for `cartservice` with Memorystore (redis).
 
 Important notes:
-- You can connect to a Memorystore (redis) instance from GKE clusters that are in the same region and use the same network as your instance.
-- You cannot connect to a Memorystore (redis) instance from a GKE cluster without VPC-native/IP aliasing enabled.
+
+- You can connect to a Memorystore (redis) instance from GKE clusters that are
+  in the same region and use the same network as your instance.
+- You cannot connect to a Memorystore (redis) instance from a GKE cluster
+  without VPC-native/IP aliasing enabled.
 
 ![Architecture diagram with Memorystore](./img/memorystore.png)
 
 ## Steps
- 
+
 1. Create a GKE cluster with VPC-native/IP aliasing enabled.
```

Comment:
Will we keep the GCP instructions within the project?
I know that the initial microservices demo is from GCP, but I though the idea behind the opentelemetry-demo-webstore was to be a totally (as much as possible) vendor agnostic project.